core/modules/query/QueryTemplate.h

```diff
@@ -127,6 +127,12 @@ class QueryTemplate {
 
     QueryTemplate(SetAliasMode aliasMode) : _aliasMode(aliasMode) {}
 
+    // set if the output should quote identifiers or not
+    void quoteIdentifiers(bool quoteIdentifiers) { _quoteIdentifiers = quoteIdentifiers; };
```

Comment:
This way of overloading methods looks a bit confusing. I would either name this particular overload as (my favorite option):
```
void setQuteIdentifiers(bool quoteIdentifiers) { ... }
```
or let it to return a value:
```
bool quoteIdentifiers(bool quoteIdentifiers) { ... }
```